messaging/about.md

```diff
@@ -25,9 +25,9 @@ There are a few concepts that are important to understand how Bandwidth's new me
 
 ## Message Storage IE `GET /messages` {#manage-storage}
 
-Messaging 2.0 does not keep _any_ records to fetch later.  If you need to keep track of delivered, error-ed, received messages after you receive the corresponding [callback event](events/messageEvents.md), you **MUST** store the events in the data-store of your choice.
+Messaging 2.0 does not keep _any_ records to fetch later.  If you need to keep track of delivered, error-ed, received messages after you receive the corresponding callback event, you **MUST** store the events in the data-store of your choice.
```

Comment:
Let's just say: "Bandwidth Messaging"

Reply:
Good catch. Let's keep an eye out for more occurrences of these